### DISCUSSION TITLE
Novos canais de Tv (SIC E OUTROS)

### DISCUSSION THREAD

**Discussion by cportugalr** [LITUATUI/M3UPT#58] [General]
Na opto existe os canais SIC ALTA DEFINIÇÃO e o canal SIC REPLAY.
Valerá a pena colocar na lista? 

Os canais Kuriakos tv, Kuriakos kids dão muito conteúdo em Português de Portugal, (depois ainda temos o Kuriakos Cine e Kuriakos music). 
Valerá a pena também colocar estes também?

**Comment by LITUATUI** [comment]
Bom dia.

Acabei de adicionar os novos canais da SIC. Quanto aos canais Kuriakos, são propaganda religiosa, não vejo interesse.

**Comment by ugly1** [comment]
SIC tem 2 horas de atraso em relação à emissão em directo: Muito obrigado pelo excelente trabalho
